src/redux/actions/samples/saveSamples.js

```diff
@@ -15,21 +16,29 @@ const saveSamples = (
 
   let payload;
 
+  const newSampleCopy = _.cloneDeep(newSample);
+
   // add new sample to payload
   if (addSample) {
     const projectSamples = projects[projectUuid].samples.reduce((acc, sampleId) => {
       acc[sampleId] = samples[sampleId];
       return acc;
     }, {});
 
+    // Do not save the fileObject to DynamoDB as it can not be serialized
+    Object.keys(newSampleCopy.files).forEach((file) => {
+      delete newSampleCopy.files[file].fileObject;
```

Comment:
We decided to disable retry button when the file is fetched from dynamo (because then we don't have fileObject to perform the retry).

Reply:
added